Makefile

```diff
@@ -35,37 +35,51 @@ verify: ## Verify code. Includes dependencies, linting, formatting, etc
 	go fmt ./...
 	golangci-lint run
 
+helm: ## Generate Helm Chart
+	cp $(RELEASE_MANIFEST) charts/karpenter/templates
+	yq w -i charts/karpenter/Chart.yaml version $(RELEASE_VERSION)
+	cd charts; helm package karpenter; helm repo index .
+
 codegen: ## Generate code. Must be run if changes are made to ./pkg/apis/...
 	controller-gen \
 		object:headerFile="hack/boilerplate.go.txt" \
 		webhook \
 		crd:trivialVersions=false \
 		paths="./pkg/..." \
-		output:crd:artifacts:config=config/crd/bases \
-		output:webhook:artifacts:config=config/webhook
+		output:crd:artifacts:config=config \
+		output:webhook:artifacts:config=config
 
 	./hack/boilerplate.sh
 
+	mv config/provisioning.karpenter.sh_provisioners.yaml config/provisioner-crd.yaml
+	mv config/manifests.yaml config/webhooks.yaml
```

Comment:
I think I'm a bit confused here - why is `config/webhooks.yaml` checked in if it's overwritten by `config/manifests.yaml`?

Reply:
It should be the opposite. I'm intending to check in config/webhooks.yaml and not config/manifests.yaml. Manifests.yaml is the hardcoded filename output of the webhook code generator.